src/main/java/com/thealgorithms/others/ArrayLeftRotation.java

```diff
@@ -0,0 +1,67 @@
+package com.thealgorithms.others;
+
+import java.util.Scanner;
+
+/*
+ * A left rotation operation on an array
+ * shifts each of the array's elements
+ * given integer n unit to the left.
+ * 
+ * @author sangin-lee 
+ */
+
+public class ArrayLeftRotation {
+
+	/*
+	 * Returns the result of left rotation of given array arr and integer n
+	 * 
+	 * @param arr : int[] given array
+	 * 
+	 * @param n : int given integer
+	 * 
+	 * @return : int[] result of left rotation
+	 */
+	public static int[] rotateLeft(int[] arr, int n) {
+		int size = arr.length;
+		int[] dst = new int[size];
+		n = n % size;
+		for (int i = 0; i < size; i++) {
+			dst[i] = arr[n];
+			n = (n + 1) % size;
+		}
+		return dst;
+	}
+
+	public static void printIntegerArray(int[] arr) {
+		int size = arr.length;
+		String str = "[ ";
+		for (int i = 0; i < size; i++) {
+			str += arr[i] + " ";
+		}
+		str += "]";
+		System.out.println(str);
+	}
+
+	public static void main(String[] args) {
```

Comment:
Please also remove main() and write a couple of tests instead as in https://github.com/TheAlgorithms/Java/blob/66d8d51de67fd3e7c7249930f4a912ad9560d823/src/test/java/com/thealgorithms/searches/QuickSelectTest.java

Reply:
@siriak I wrote some junit tests, but I have no idea of why build fails.

> package org.junit.jupiter.api does not exist

I wrote the test just like QuickSelectTest.java you've commented.
Can you give me some advice?

Reply:
I resolved it. Thanks 👍